HousingSearchApi/V1/Controllers/GetAssetRelationshipController.cs

```diff
@@ -0,0 +1,47 @@
+using Amazon.Lambda.Core;
+using Hackney.Core.Logging;
+using HousingSearchApi.V1.Boundary.Requests;
+using HousingSearchApi.V1.Boundary.Responses;
+using HousingSearchApi.V1.Boundary.Responses.Metadata;
+using HousingSearchApi.V1.UseCase.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Threading.Tasks;
+
+namespace HousingSearchApi.V1.Controllers
+{
+    [ApiVersion("1")]
+    [Produces("application/json")]
+    [Route("api/v1/search/assetrelationships")]
+    [ApiController]
+    public class GetAssetRelationshipController : BaseController
+    {
+        private readonly IGetAssetRelationshipsUseCase _getAssetRelationshipsUseCase;
+
+        public GetAssetRelationshipController(IGetAssetRelationshipsUseCase getAssetRelationshipsUseCase)
+        {
+            _getAssetRelationshipsUseCase = getAssetRelationshipsUseCase;
+        }
+
+        [ProducesResponseType(typeof(APIResponse<GetAssetRelationshipsResponse>), 200)]
+        [ProducesResponseType(typeof(APIResponse<NotFoundException>), 404)]
```

Comment:
Did you intend to return a NotFoundResponse?

Reply:
Hi mate - I'm in the process of adding validation - it's not ready for review yet 👍

Reply:
Just posted it to give Angelo the likely frontend contract

Reply:
I had my suspicions.